docs/interaction/rpc.md

```diff
@@ -10,118 +10,508 @@ The following methods are available:
 
 ## Status
 
-`status` returns current status of the node: `http post http://127.0.0.1:3030/ jsonrpc=2.0 method=status params:="[]" id="dontcare"`
+`status`
 
-Result:
+Returns current status of the node
 
-```javascript
+- HTTP `POST`
+- URL `https://rpc.nearprotocol.com`
+- method `query`
+- params *none*
+
+```bash
+http post https://rpc.nearprotocol.com jsonrpc=2.0 method=status params:='[]' id=dontcare
+```
+
+```json
 {
-    "id": "dontcare",
-    "jsonrpc": "2.0",
-    "result": {
-        "chain_id": "test-chain-nmZGf",
-        "rpc_addr": "0.0.0.0:3030",
-        "sync_info": {
-            "latest_block_hash": "ugvtXLvad6DMGIYOf/NpgHt2AWbnhooTH53kp2GwB+w=",
-            "latest_block_height": 17034,
-            "latest_block_time": "2019-05-26T04:50:11.150214Z",
-            "latest_state_root": "PHvWhgQjY37IOAiaUgnpDiZKwlcfzu+c585hGuIS5Qo=",
-            "syncing": false
-        }
+  "id": "dontcare",
+  "jsonrpc": "2.0",
+  "result": {
+    "chain_id": "testnet",
+    "rpc_addr": "0.0.0.0:8080",
+    "sync_info": {
+      "latest_block_hash": "En24v7abQGFMRAeqmA7742z9QUN1ZGvgPyD3BsVpj5fp",
+      "latest_block_height": 289032,
+      "latest_block_time": "2020-01-23T15:11:41.594468356Z",
+      "latest_state_root": "8tJ13zjE6ZRj3CwBdAgbv7fH3Nr4SPaoYygNbqKCReMa",
+      "syncing": false
+    },
+    "validators": [
+      {
+        "account_id": "far",
+        "is_slashed": false
+      }
+    ],
+    "version": {
+      "build": "1a0ea717-modified",
+      "version": "0.4.12"
     }
+  }
 }
 ```
 
-## Send transaction \(async\)
+## Send transaction (async)
+
+`broadcast_tx_async`
 
-`broadcast_tx_async`: sends transaction and returns right away with the hash of the transaction in base58.
+Sends transaction and returns right away with the hash of the transaction in base58.
 
-`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=broadcast_tx_async params:="[<base 64 of the SignedTransaction>]" id="dontcare"`
+- HTTP `POST`
+- URL `https://rpc.nearprotocol.com`
+- method `broadcast_tx_async`
+- params
+  - (1) `<base 64 of the SignedTransaction>`
+
+```bash
+http post https://rpc.nearprotocol.com jsonrpc=2.0 method=broadcast_tx_async params:='["<base 64 of the SignedTransaction>"]' id=dontcare
+```
```

Comment:
let's add example output here as well.

Reply:
✅ on this one

also fixed incorrect `broadcast_tx_commit` output entry since that's what NEAR Shell uses when we call a function anyway.